.ado/azure-pipelines.yml

```diff
@@ -77,7 +77,7 @@ jobs:
   - job: ApplePR
     displayName: Apple PR
     pool:
-      vmImage: 'macos-11'
+      vmImage: 'internal-macos12'
```

Comment:
What is this "internal" image? And does it by chance have SIP disabled? :D

Reply:
what is SIP?

Reply:
[SA](https://developer.apple.com/documentation/security/disabling_and_enabling_system_integrity_protection?language=objc)

It prevents us from running automation tests on macOS, and is not disabled on public ADO pools

Reply:
but are there actually automation test that are running right now? until we actually have the latest version of xcode in the public image we kind of have to use the internal.